flowtools/upload_files/.shed.yml

```diff
@@ -0,0 +1,15 @@
+owner: Cristel Thomas
+name: upload_files
+description: upload files including those with an fcs extension
+long_description: |
+    **Input**
+    One or more files
+    
+    **Output**
+    Files loaded in history 
+    
+    https://www.immportgalaxy.org
+categories:
+    - Flow Cytometry Analysis
+remote_repository_url: https://github.com/ImmPortDB/immport-galaxy/tree/master/tools/data_source/
```

Comment:
Since we won't be merging into theirs, I would suggest that we use here the URL to this git repo, which will hold the source of this tool.

Reply:
Thanks Pablo I will do that form now on. @pcm32

Reply:
@pcm32 I was just wondering if it is easier for me to make any changes after you review the PRs and then push them again or will you make the changes?